apps/synapse-portal-framework/src/utils/PortalsDoiSerializer.test.ts

```diff
@@ -0,0 +1,79 @@
+import PortalsDoiIdSerializer from '@/utils/PortalsDoiIdSerializer'
```

Comment:
does this not need to be a .vitest.ts file?

Reply:
We only need to do that in synapse-react-client